tests/spread/core24/grammar/snap/snapcraft.yaml

```diff
@@ -15,6 +15,12 @@ platforms:
     build-on: amd64
     build-for: arm64
 
+build-packages:
+  - on amd64:
+    - libogg-dev
+  - on arm64:
```

Comment:
Should we test the `to arch` grammar here too?

Reply:
Good call, updated!